src/js/login/containers/Main.jsx

```diff
@@ -145,12 +145,14 @@ class Main extends React.Component {
   checkTokenStatus() {
     if (sessionStorage.userToken) {
       if (moment() > moment(sessionStorage.entryTime).add(SESSION_REFRESH_INTERVAL_MINUTES, 'm')) {
-        // TODO(crew): make more customized prompt.
-        if (confirm('For security, you’ll be automatically signed out in 2 minutes. To stay signed in, click OK.')) {
-          this.handleLogin();
-        } else {
-          this.handleLogout();
-        }
+
```

Comment:
What happens if the entryTime is older than 45 minutes now? Do the login links show up?